### DISCUSSION TITLE
How to quantize a model?

### DISCUSSION THREAD

**Discussion by yukiarimo** [coqui-ai/TTS#3820] [General Q&A]
Hello! I’ve seen LLaMA and even Whisper models quantized in GGUF/GGML. Is it possible to quantize Coqui XTTS-V2 model?

**Comment by bibhabasumohapatra** [comment]
@yukiarimo did you get an answer?

**Comment by yukiarimo** [comment]
No, thanks! Developed custom TTS. Closing
